src/broker/module.c

```diff
@@ -151,14 +151,13 @@ static int module_finalizing (module_t *p)
     flux_future_t *f;
 
     if (!(f = flux_rpc_pack (p->h_module_end,
-                             "broker.module-status",
+                             "module.status",
```

Comment:
Will the service name changes have any unintended consequences with version compatibility. (e..g perhaps when using `flux proxy`)? I can't think of much, but thought I'd bring it up just in case.

Reply:
I was assuming that for the most part people will not be running module commands through flux-proxy, and flux-proxy does make you force the connection if the versions are different, so maybe OK?

Oh that reminds me that the original topic strings are mentioned in [RFC 5](https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_5.html), so I'll follow this with an rfc pr.

I don't think there is too much danger of this causing problems.